packages/cli/src/reporters/list.ts

```diff
@@ -24,12 +24,12 @@ export default class ListReporter extends AbstractListReporter {
     this._clearSummary()
     if (checkResult.hasFailures) {
       // Print the failed check result above the status section
-      console.log(formatCheckTitle(CheckStatus.FAILED, checkResult))
+      stdOutWriteLn(formatCheckTitle(CheckStatus.FAILED, checkResult))
     }
     if (this.verbose || checkResult.hasFailures) {
-      console.log('')
-      console.log(indentString(formatCheckResult(checkResult), 4))
-      console.log('')
+      stdOutWriteLn('')
+      stdOutWriteLn(indentString(formatCheckResult(checkResult), 4))
+      stdOutWriteLn('')
```

Comment:
This can also use your new functions
```suggestion
      stdOutWriteLn(indentString(formatCheckResult(checkResult), 4), 2, 1)
```